app.js

```diff
@@ -13,324 +13,287 @@ if (!supportsLocalStorage()) {
       }
-    }
+    });
+  } else if (!eventTarget.checked) {
+    fullInviteList.map((current, index) => {
+      renderedInviteList.push(current);
+    });
   }
+  reloadDOM();
+  };
 
-  //Show all li labels
-  function showAllLabels() {
-    const lis = ul.children;
-    for (i = 0 ; i < lis.length ; i++) {
-      let li = lis[i];
-      let attending = li.firstElementChild.nextElementSibling.firstElementChild;
-      let attendingParent = attending.parentNode;
-      let nextLabel = attendingParent.nextElementSibling;
-      attendingParent.style.display = '';
-      nextLabel.style.display = '';
-    }
+  //re-maps renderedInviteList to include invitees who have not responded. Or to include all invitees, if filter is unchecked
+  const filterTheWorstKindOfPeople = (eventTarget) => {
+  renderedInviteList = [];
+  if (eventTarget.checked) {
+    fullInviteList.map((current, index) => {
+      if ((current.attending === false) && (current.notAttending === false)) {
+        renderedInviteList.push(current);
+      }
+    });
+  } else if (!eventTarget.checked) {
+    fullInviteList.map((current, index) => {
+      renderedInviteList.push(current);
+    });
   }
+  reloadDOM();
+  };
 
-  //Create new list items replete w/ invitee name, cofirmed checkbox, and edit/remove buttons
-  function createLI(text) {
-    //Create new elements.
-    function createElement(elementType, propertyName, propertyValue) {
-      const element = document.createElement(elementType);
-      element[propertyName] = propertyValue;
-      return element;
-    }
-    //Create and append element to the new li. See createElement() for argument expectations.
-    function appendToLI(elementType, propertyName, propertyValue) {
-      const element = createElement(elementType, propertyName, propertyValue);
-      li.appendChild(element);
-      return element;
-    }
-    const li = document.createElement('li');
-    appendToLI('span', 'textContent', text)
-      .className = 'name';
-    appendToLI('label', 'textContent', 'Attending')
-      .appendChild(createElement('input', 'type', 'checkbox'));
-    appendToLI('label', 'textContent', 'Not Attending')
-      .appendChild(createElement('input', 'type', 'checkbox'));
-    appendToLI('span', 'textContent', 'Notes...')
-      .className = 'notes';
-    appendToLI('button', 'textContent', 'Edit');
-    appendToLI('button', 'textContent', 'Remove');
-    return li;
+  //Switches state between editing and saved modes, coordinates info that needs to pass through the state change. Syncs edits with all lists.
+  const editNotes = (invitee, eventTarget) => {
+    fullInviteList.map((current, index) => {
+      let input = eventTarget.previousElementSibling.firstElementChild.value;
+      if (invitee === current.name.replace(/\s/g, '') && current.editState === 'Saved') {
+        current.editState = 'Editing';
+      } else if (invitee === current.name.replace(/\s/g, '') && current.editState === 'Editing') {
```

Comment:
Since you're calling this variable more than once in this map, you should define it as a constant at the top of the map and then call it like:
```
const name = trim(current.name);
...
if (invitee === name...
```